packages/apidom-ls/src/config/openapi/path-template/lint/value--valid.ts

```diff
@@ -0,0 +1,23 @@
+import { DiagnosticSeverity } from 'vscode-languageserver-types';
+
+import ApilintCodes from '../../../codes';
+import { LinterMeta } from '../../../../apidom-language-types';
+import { OpenAPI } from '../../target-specs';
+
+const valueValidLint: LinterMeta = {
+  code: ApilintCodes.OPENAPI2_PATH_TEMPLATE_VALUE_VALID,
+  source: 'apilint',
+  message: 'path template expressions is not matched with Parameter Object(s)',
+  severity: DiagnosticSeverity.Error,
+  linterFunction: 'apilintOpenAPIPathTemplateValid',
+  marker: 'value',
+  targetSpecs: OpenAPI,
+  conditions: [
```

Comment:
We want to validate matching of Paramater Objects only when the path template is well formed and contains at least one `template expression`. That is achieved by `params: [true]`.